packages/merge-styles/tslint.json

```diff
@@ -1,4 +1,6 @@
 {
   "extends": ["office-ui-fabric-react-tslint"],
-  "rules": {}
+  "rules": {
+    "no-any": false
```

Comment:
I think we could just do `tslint:disable:no-any` like the other places

Reply:
My POV is that this library in particular has a high density of real, unavoidable uses of any (the implementation function in the overloads, for instance). Having a ton of tslint:disable scattered around is a big mess.

We still disallow implicit anys, and any use of `any` will be scrutinized.